mp/src/game/server/momentum/mom_player.cpp

```diff
@@ -1753,11 +1757,66 @@ void CMomentumPlayer::TimerCommand_Reset()
             // MOM_TODO do a trace downwards from the top of the trigger's center to touchable land, teleport the player there
             Teleport(&pCurrentZone->WorldSpaceCenter(), nullptr, &vec3_origin);
         }
+        return;
+    }
+
+    const Vector *pVec = nullptr;
+    const QAngle *pAng = nullptr;
+
+    if (stage == 1)
```

Comment:
Use `TimerCommand_Restart` for the case when they want to restart the track, for less code duplication.

Reply:
Did this, though had to move the safeguard to inside the `mom_restart` cmd in `mom_timer.cpp`.